ui/src/Roles/RoleEditRequest.interface.ts

```diff
@@ -15,7 +15,8 @@
  * limitations under the License.
  */
 
-export interface LocationEditRequest {
-    id: number;
-    updatedName: string;
-}
\ No newline at end of file
+import {TagRequest} from '../Tags/TagRequest.interface';
+
+export interface RoleEditRequest extends TagRequest {
+    colorId?: number;
+}
```

Comment:
Is this necessary if the RoleEditRequest and RoleAddRequest are the same thing?

Reply:
no it is not. But we decided to stop and make the objects another refactor